ServerCore/Pages/Resources/PH21New/RulesPartial.cshtml

```diff
@@ -7,15 +7,24 @@
         <li id="WhatToBring">
             The more prepared you are for Puzzlehunt weekend, the better time you'll have participating in it. Here's a list of supplies that you may find helpful:
             <ul>
-                <li>
-                    Office supplies: Pens, pencils, colored pencils, graph paper, scissors,
-                    clear tape, ruler/straight edge, protractor, highlighters, glue stick, compass
+                <li>Most puzzles can be solved without printing, but a few will require a printer (ideally color), as well as scissors and tape.</li>
+                <li>If you're interested in earning hints to be applied to other puzzles, you will want either or both of:
+                    <ul>
+                        <li>Several smartphones or other devices capable of taking still pictures and videos and uploading them to the Internet</li>
+                        <li>One or more computers or consoles with Minecraft installed. Any edition/version will do, but do note that the XBox version is free for Game Pass subscribers, and <b>Microsoft FTEs</b> can get a <a href="https://studiostokens.azurewebsites.net/">free Game Pass subscription</a>.</li>
```

Comment:
This will nav the current page. Would `target="_blank"`

Reply:
Good catch, I always forget